modules/deploy/pages/redpanda/manual/resilience/shadowing.adoc

```diff
@@ -0,0 +1,677 @@
+  auto_create_shadow_topic_filters:      # Filters for automatic topic creation
+    - pattern_type: "PREFIX"             # Match topics starting with this prefix
+      filter_type: "INCLUDE"
+      name: "<TOPIC_PREFIX>"             # Examples: "prod-", "production-", "app-"
+    - pattern_type: "LITERAL"            # Exclude specific topics
+      filter_type: "EXCLUDE"
+      name: "<EXCLUDED_TOPIC>"           # Examples: "temp-data", "test-topic"
+    - pattern_type: "LITERAL"            # Include all remaining topics (wildcard)
+      filter_type: "INCLUDE"
+      name: "*"
+  synced_shadow_topic_properties:        # Additional topic properties to sync (beyond defaults)
+    - "retention.ms"                     # Topic retention time
+    - "segment.ms"                       # Segment roll time
+    - "<CUSTOM_PROPERTY>"                # Examples: "compression.type", "min.insync.replicas"
+
+consumer_offset_sync_options:
+  interval: "30s"                        # How often to sync consumer group offsets
+  enabled: true                          # Enable consumer offset synchronization
+  group_filters:                         # Filters for consumer groups to sync
+    - pattern_type: "PREFIX"
+      filter_type: "INCLUDE"
+      name: "<CONSUMER_GROUP_PREFIX>"    # Examples: "prod-consumer-", "app-"
+    - pattern_type: "LITERAL"
+      filter_type: "EXCLUDE"
+      name: "<EXCLUDED_GROUP>"           # Examples: "test-consumer", "debug-group"
+
+security_sync_options:
+  interval: "30s"                        # How often to sync security settings
+  enabled: true                          # Enable security settings synchronization
+  role_filters:                          # Filters for user roles to sync
+    - pattern_type: "LITERAL"
+      filter_type: "INCLUDE"
+      name: "<ROLE_NAME>"                # Examples: "application-user", "read-only"
+  scram_cred_filters:                    # Filters for SCRAM credentials to sync
+    - pattern_type: "PREFIX"
+      filter_type: "EXCLUDE"
+      name: "<ADMIN_PREFIX>"             # Examples: "admin-", "root-"
+  acl_filters:                           # Filters for ACLs to sync
+    - resource_filter:
+        resource_type: "TOPIC"           # Resource type: "TOPIC", "GROUP", "CLUSTER"
+        pattern_type: "PREFIX"           # Pattern type: "LITERAL", "PREFIX", "PREFIXED"
+        name: "<RESOURCE_PATTERN>"       # Examples: "prod-", "app-data-"
+      access_filter:
+        principal: "User:<USERNAME>"     # Principal name, example: "User:app-service"
+        operation: "READ"                # Operation: "READ", "WRITE", "CREATE", "DELETE", "ALTER", "DESCRIBE", "ANY"
+        permission_type: "ALLOW"         # Permission: "ALLOW" or "DENY"
+        host: "<HOST_PATTERN>"           # Host pattern, examples: "*", "10.0.0.0/8", "app-server.example.com"
+----
+====
+
+See also: link:/api/doc/admin/[Admin API v2 reference^]
+
+== Failover
```

Comment:
Add your note from the beginning of the doc here--users don't always read from start to finish, and we wouldn't want them to miss it:

Experiencing an active disaster? See [Emergency Shadowing Failover](https://deploy-preview-1381--redpanda-docs-preview.netlify.app/current/deploy/redpanda/manual/resilience/shadowing/#deploy:redpanda/manual/resilience/emergency-shadowing.adoc) for immediate step-by-step emergency procedures.